File: server/DAL/Models/Trip.cs
using System;
using System.Collections.Generic;

namespace DAL.Models;

public partial class Trip
{
    public int TripId { get; set; }

    public string? TripDestination { get; set; }

    public int? KindId { get; set; }

    public DateTime? TripDate { get; set; }

    public TimeSpan? TripTime { get; set; }

    public int? TripHour { get; set; }

    public int? TripAvailability { get; set; }

    public decimal? TripPrice { get; set; }

    public string? TripImage { get; set; }

    public virtual Kind? Kind { get; set; }

    public virtual ICollection<Order> Orders { get; } = new List<Order>();
}

File: client/src/app/classes/trip.ts
import { Time } from "@angular/common";

export class trip{
   
    constructor(public TripId:number, public TripDestination?:string, public KindId?:number, public TripDate?:Date, public TripTime?:Time, public TripHour?:number, public TripAvailability?:number, public TripPrice?:number, public TripImage?:string,public KindName?:string,public Paramedic?:boolean ) { }
}